src/visualizers/panels/PlotlyGraph/PlotlyGraphControl.js

```diff
@@ -0,0 +1,169 @@
+/*globals define, WebGMEGlobal*/
+
+define([
+    'js/Constants'
+], function (
+    CONSTANTS
+) {
+    'use strict';
+
+    const GRAPH = 'Graph',
+        DATA = 'data';
+
+    function PlotlyGraphControl(options) {
+
+        this._logger = options.logger.fork('Control');
+
+        this._client = options.client;
+
+        // Initialize core collections and variables
+        this._widget = options.widget;
+
+        this._embedded = options.embedded;
+        this._activeNodeMetaName = options.activeNodeMetaName || GRAPH;

```

Comment:
It would be nice if we could remove this since it tightly couples the visualizer to a metamodel (and this is already implied by setting this as the visualizer for the given node).

Reply:
That is, it should be safe to assume that `this._currentNodeId` corresponds to a graph node.